biojava-structure/src/main/java/org/biojava/nbio/structure/SubstructureIdentifier.java

```diff
@@ -135,14 +149,18 @@ public String toString() {
 	 */
 	@Override
 	public String getIdentifier() {
-		if (ranges.isEmpty()) return pdbId;
+		if (ranges.isEmpty()) return pdbId.getId();
 		return pdbId + "." + ResidueRange.toString(ranges);
 	}
 
 	public String getPdbId() {
```

Comment:
Should we deprecate this? Two methods that differ only by capitalization is confusing.

Reply:
Yes. Done.